examples/gnrc_border_router/Makefile.board.dep

```diff
@@ -7,11 +7,16 @@ ifeq (,$(filter native,$(BOARD)))
   else ifeq (wifi,$(UPLINK))
     ifneq (,$(filter esp32 esp8266,$(CPU)))
       USEMODULE += esp_wifi
-      USEMODULE += esp_now
+      ifneq (ble, $(DOWNLINK))
+        USEMODULE += esp_now
+      endif
     else
       $(error Only esp32 and esp8266 are currently supported)
     endif
   endif
+  ifeq (ble, $(DOWNLINK))
+    USEMODULE += nimble_rpble
```

Comment:
Why do we need RPL for that?

Reply:
Hm I was under the impression that RPL gets selected by the application anyway, but turns out this is not the case.